tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-order-mapper-test.php

```
@@ -1426,13 +1436,77 @@ public function test_shipping_throws_when_no_matching_rate() {
+			$shipping_item = reset( $shipping_items );
+			$this->assertEquals( 'Throwing Method', $shipping_item->get_method_title() );
+			$this->assertEquals( 'stripe_agentic', $shipping_item->get_method_id() );
+			$this->assertEqualsWithDelta( 9.99, (float) $shipping_item->get_total(), 0.001 );
+
+			$order->delete( true );
+		} finally {
+			remove_filter( 'woocommerce_package_rates', $thrower );
+			\WC_Cache_Helper::get_transient_version( 'shipping', true );
+			WC()->shipping()->reset_shipping();
 		}
 	}
```

PHPMD (2.15.0)

[error] The method test_shipping_falls_back_to_free_form_line_when_calculation_throws is not named in camelCase. (undefined)

(CamelCaseMethodName)